cmake/packaging.cmake

```diff
@@ -139,9 +139,9 @@ set(CPACK_DEBIAN_${IRODS_PACKAGE_COMPONENT_SERVER_NAME_UPPERCASE}_PACKAGE_CONTRO
 set(CPACK_RPM_${IRODS_PACKAGE_COMPONENT_SERVER_NAME}_PACKAGE_NAME "irods-server")
 set(CPACK_RPM_${IRODS_PACKAGE_COMPONENT_SERVER_NAME}_PACKAGE_REQUIRES "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime = ${CPACK_RPM_PACKAGE_VERSION}-${CPACK_RPM_PACKAGE_RELEASE}, openssl, python3, python3-psutil, libarchive, chkconfig, glibc, libgcc, libstdc++")
 if (IRODS_LINUX_DISTRIBUTION_VERSION_MAJOR EQUAL 7)
```

Comment:
Can't we remove this since we don't support EL7 anymore?

Reply:
I'm sure we can, but I'm going to leave that to a separate PR so that this one gets in quicker.

Reply:
Okay. Let's just leave it, then.